core/src/main/java/com/netflix/iceberg/util/BinPacking.java

```diff
@@ -144,6 +148,10 @@ public void add(T item, long weight) {
         this.binWeight += weight;
         items.add(item);
       }
+
+      public long getWeight() {
```

Comment:
The convention used in Iceberg is to drop the `get` from getter method names so that they look normal when called from other languages, like Scala, that don't have this verbose name convention. So this should just be `weight`.

Reply:
+1 thanks.